R/checks.R

```diff
@@ -207,6 +207,32 @@ check_form4.1_box5 <- function(form) {
                         "Textfield and coding box 5 of form4.1 is completed if and only if the other option is chosen"))
 }
 
+#' Filters out invalid rows for box 6 of form 4.1 for location of first administration = At The Injury Scene
+#' 
+#' @param form a form containing ptstatus and form 4.1 
+#' 
+#' @return a data frame containing all the invalid rows
+#' 
+#' @import tidyverse
+#' @export
+check_form4.1_box6 <- function(form){
```

Comment:
You are not checking the coding box 6 of form4.1. You are checking antibiotics administration (abx). A name like check_abx_locabx_1 would be more appropriate.